doc/api/errors.md

```diff
@@ -1362,6 +1362,11 @@ An attempt was made to send an unsupported "handle" over an IPC communication
 channel to a child process. See [`subprocess.send()`][] and [`process.send()`][]
 for more information.
 
+<a id="ERR_INVALID_HOST_PORT_COMBO"></a>
+### `ERR_INVALID_HOST_PORT_COMBO`
+
+The provided path is not a valid host:port combo.
+
```

Comment:
The existing `ERR_INVALID_ARG_VALUE` error code should be used instead of adding a new one